src/foam/u2/view/UnstyledTableView.js

```diff
@@ -319,7 +311,12 @@ foam.CLASS({
 
     async function initE() {
       var view = this;
-
+      const asyncRes = await this.filterUnpermited(view.of.getAxiomsByClass(foam.core.Property));
+      this.allColumns = ! view.of ? [] : [].concat(
+        asyncRes.map(a => a.name),
+        view.of.getAxiomsByClass(foam.core.Action)
+        .map(a => a.name)
+      );
```

Comment:
please add
`
//to filter properties values of which displayed in columns

const asyncProps = await this.filterUnpermited(this.props.map(p => p.property));

this.props = asyncProps;
`